api/src/models/userCountryEdits.js

```diff
@@ -1,9 +1,4 @@
 const db = require('../db/connection')
-const util = require('util')
```

Comment:
This is needed in the clocks file for users. It either has to be added to user clocks or be restored here.